Update the website for the release of POB2

Paliak opened the pull request:
This pr makes the following changes:
- Update landing page to include overview image and link to POB2
- Change github icons to POE icons on download buttons
- Does various miscellaneous tweaks to content scaling and positioning
- ~~Removes Privacy and Imprint as i think they're unnecessary~~
- Updated main images
- Updates development container and readme
- Moves and slightly updates the paragraph mentioning history of POB Community Fork

A lot of the changes are subjective so I'm to discussion.

Paliak committed:
FEAT: update the website for the release of POB2

ppoelzl commented:
The privacy and imprint pages are a legal requirement for me to host the website. Please do not change them.

Wires77 reviewed (commented):
~~Running this with the Docker compose file grants me this error:~~
```
jekyll-1  | Resolving dependencies...
jekyll-1  | listen-3.2.1 requires ruby version >= 2.2.7, ~> 2.2, which is incompatible with
jekyll-1  | the current version, ruby 3.1.1p18
```

~~I assume the issue is with the Docker base image.  We were using `webrick` to be compatible with Ruby 3+, not sure if that's still needed to run this or if something is hosed on my setup.~~

Turns out going through the Ruby/bundler update song and dance fixed this, which was surprising, seeing as this should just be contained in the Docker container provided by Jekyll...

Wires77 commented on index.html:
```diff
@@ -2,23 +2,40 @@
+				<div class="cta button alt">
+					<a href="https://github.com/PathOfBuildingCommunity/PathOfBuilding-PoE2/releases/latest/download/PathOfBuildingCommunity-PoE2-Setup.exe ">
+						<img class="social-icon" src="{{ site.baseurl }}/images/poe2_icon.png" alt="Path of Exile 1 logo"/>
+						Download POB2 for POE2!
```
Change these to not use acronyms
```suggestion
						Download for Path of Exile 2!
```

Paliak replied:
Added in https://github.com/PathOfBuildingCommunity/pathofbuildingcommunity.github.io/pull/19/commits/e235590cfb39d395101e2fbdc4768b3390504c71

Wires77 replied:
Oh, I actually meant just remove the "Path of Building [2]" altogether.  It gets a little cramped and wordy with both terms there, and I think the "Path of Building" is redundant considering it's at the top of the page (see original website how it doesn't have it either)

Paliak replied:
I'll make it just `Download for Path of Exile [2]`. Local had some concerns regarding clarity of which button goes to which version so i wanted to make it clear.

Wires77 replied:
I think now that we have the icon of the game itself it's a little clearer which is which

Wires77 commented on _sass/footer.scss:
```diff
@@ -103,10 +102,7 @@ footer {
-		&:hover {
-			color: #fff;
-		}
+		color: #fff;
```
This change seemed to make the links in the footer not highlight on hover anymore.

Paliak replied:
My issue with the links is that they didn't really stand out. As in you wouldn't know that the "here" was actually a link. Some other links didn't change in any way on hover either. I modified it further in https://github.com/PathOfBuildingCommunity/pathofbuildingcommunity.github.io/pull/19/commits/e235590cfb39d395101e2fbdc4768b3390504c71 to make all links in the body of the page get an underline when hovered over. If they're in the middle of a sentence i made them bold to make them stand out a bit more.

Wires77 replied:
This is that link in Firefox and Chrome on prod when hovered:
![image](https://github.com/user-attachments/assets/a4b508c6-23fd-48aa-b8dd-6ce07373f535)

Looked exactly the same in this PR when I tested it yesterday too.  Other links behave the same, other than the ones I mentioned at the bottom.  Is that not the case for you?  Every link does something on hover for me.

Paliak replied:
Ahh. The `See all releases` didn't change so i focused on fixing that and in the process i guess i broke the other ones as well and thought they needed fixing. I'll revert most of the changes. Current prod has them underlined always. Thoughts on making them bold on hover? Or just making them get the underline only on hover like the other ones?

Wires77 replied:
Latest commit looks good, thanks for the tweaks!

Paliak committed:
fix: restore privacy.md, imprint.md

Paliak committed:
fix: misc scaling issues especially on chrome

Paliak committed:
fix: update Gemfiles

I understand that the README says to not commit Gemfile file but having
to manually boundle update everytime will be very annoying. This way the
docker container should work oob.

Paliak commented:
> The privacy and imprint pages are a legal requirement for me to host the website. Please do not change them.

Reverted back in https://github.com/PathOfBuildingCommunity/pathofbuildingcommunity.github.io/pull/19/commits/1fe736433a39671fe9e7f2bbbb838aa374f27fb3.

Not sure i understand the need for that though. I assume this is hosted on github pages?

Paliak commented:
> ~Running this with the Docker compose file grants me this error:~
> 
> ```
> jekyll-1  | Resolving dependencies...
> jekyll-1  | listen-3.2.1 requires ruby version >= 2.2.7, ~> 2.2, which is incompatible with
> jekyll-1  | the current version, ruby 3.1.1p18
> ```
> 
> ~I assume the issue is with the Docker base image. We were using `webrick` to be compatible with Ruby 3+, not sure if that's still needed to run this or if something is hosed on my setup.~
> 
> Turns out going through the Ruby/bundler update song and dance fixed this, which was surprising, seeing as this should just be contained in the Docker container provided by Jekyll...

Yeah i wasn't sure whether to update the Gemfile.lock file. I had it updated locally.

I pushed it in https://github.com/PathOfBuildingCommunity/pathofbuildingcommunity.github.io/pull/19/commits/1ad86399d2adccdd1a93f5726f2c278a7989d76e to make the container work OOB.

Wires77 commented:
> > The privacy and imprint pages are a legal requirement for me to host the website. Please do not change them.
> 
> Reverted back in [1fe7364](https://github.com/PathOfBuildingCommunity/pathofbuildingcommunity.github.io/commit/1fe736433a39671fe9e7f2bbbb838aa374f27fb3).
> 
> Not sure i understand the need for that though. I assume this is hosted on github pages?

It's hosted by GitHub Pages, but the domain name is registered under [ppoelzl](https://github.com/ppoelzl)'s name and these pages are required for websites in his country.

Paliak committed:
fix: shorten download links. revert back the blue links

Paliak commented:
Screenshot of current state as of [d936a1c](https://github.com/PathOfBuildingCommunity/pathofbuildingcommunity.github.io/pull/19/commits/d936a1cd9edac8950e29c2868f94a44ed85150a0) in firefox

![obraz](https://github.com/user-attachments/assets/d295d428-b96e-4f81-84a1-8c26531a29fc)
